### PR TITLE
Inconsistency with timestamps

### PR DESCRIPTION
**Pull request by nplasterer**
Fixes the date issue https://github.com/xmtp/xmtp-react-native/issues/394 and the membership list.

Also makes some better naming for consent items

### CODE REVIEW

File: library/src/main/java/org/xmtp/android/library/libxmtp/MessageV3.kt [hunk 1]
```diff
@@ -80,7 +80,7 @@ data class MessageV3(val client: Client, private val libXMTPMessage: FfiMessage)
             topic = Topic.groupMessage(convoId.toHex()).description,
             encodedContent = decode().encodedContent,
             senderAddress = senderInboxId,
-            sentAt = Date(),
+            sentAt = sentAt,
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/libxmtp/MessageV3.kt:83]: This was the fix for the timestamp issues 🤦‍♀️

File: library/src/main/java/org/xmtp/android/library/Contacts.kt [hunk 2]
```diff
@@ -304,14 +303,14 @@ data class Contacts(
         consentList.publish(entries)
     }
 
-    suspend fun allowInboxId(inboxIds: List<String>) {
+    suspend fun allowInbox(inboxIds: List<String>) {
```
Comment (alexrisch) [inline library/src/main/java/org/xmtp/android/library/Contacts.kt:306]:
```suggestion
    suspend fun allowInboxes(inboxIds: List<String>) {
```

File: library/src/main/java/org/xmtp/android/library/Contacts.kt [hunk 3]
```diff
@@ -304,14 +303,14 @@ data class Contacts(
...
         val entries = inboxIds.map {
             consentList.allowInboxId(it)
         }
         consentList.publish(entries)
     }
 
-    suspend fun denyInboxIds(inboxIds: List<String>) {
+    suspend fun denyInbox(inboxIds: List<String>) {
```
Comment (alexrisch) [inline library/src/main/java/org/xmtp/android/library/Contacts.kt:313]:
```suggestion
    suspend fun denyInboxes(inboxIds: List<String>) {
```

**Review by alexrisch — APPROVED** [review]

File: library/src/main/java/org/xmtp/android/library/Contacts.kt (hunk 2 above)
Comment (nplasterer) [reply library/src/main/java/org/xmtp/android/library/Contacts.kt:306]: Ya this is what I named it in react native. But for some reason didn't make these plurals for any of the other ones (groups and addresses) 😮‍💨

**Pull request merged**